framework/include/base/MooseError.h

```diff
@@ -42,10 +42,10 @@
   {                                                                                 \
     std::ostringstream _error_oss_;                                                 \
     _error_oss_ << "\n\n"                                                           \
-                << (Moose::_color_console ? XTERM_RED : "")                         \
+                << COLOR_RED                                                        \
```

Comment:
Regardless of whether the new global gets merged, these `COLOR` changes should be.